### ISSUE TITLE
[Keyboard Navigation – BotKit]: Multiple <h1> headings are defined in the page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#1]
![MAS2 4 6_BotKit_Headings](https://user-images.githubusercontent.com/48272005/54119573-adab4800-441b-11e9-9750-e3fcfd94d311.png)
User Experience:
Users who rely on Keyboards will affect to navigate the page, if Multiple <h1> are defined.

Test Environment:
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Developer Tool bar (F 12)

Repro Steps:
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to Headings in the page.
Step 3: Open Developer Tool bar (F 12)
Step 4: Verify whether multiple <h1> headings are defined in the page or not.

Actual Result:
Multiple <h1> headings are defined in the page.
Below Heading are defined as <h1>
BotKit
BotKit Code Basics
BotKit Community Resources

Expected Result:
Single <h1> heading should be defined in the page.

MAS Reference Link:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}]

**Comment by benbrown** [comment]
This has been corrected.  

@v-gasikk can you verify?

**Comment by v-gasikk** [comment]
Issue is Fixed. Hence Closing the bug.
**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Developer Tool bar (F 12)

Now Single h1 heading is defined in the page.
